docs/spec/manifest-v2-1.md

```diff
@@ -0,0 +1,146 @@
+# Image Manifest Version 2, Schema 1 
+
+This document outlines the format of of the V2 image manifest.  Image manifests
+describe the various constituents of a docker image.  Image manifests can be
+ serialized to JSON format with the following media types:
+
+Manifest Type  | Media Type
+------------- | -------------
+manifest  | "application/vnd.docker.distribution.manifest.v1+json"
+signed manifest  | "application/vnd.docker.distribution.manifest.v1+prettyjws"
+
+*Note that "application/json" will also be accepted for schema 1.*
+
+References: 
+ 
+ - [Proposal: JSON Registry API V2.1](https://github.com/docker/docker/issues/9015) 
+ - [Proposal: Provenance step 1 - Transform images for validation and verification](https://github.com/docker/docker/issues/8093)
+
+## *Manifest* Field Descriptions
+
+Manifest provides the base accessible fields for working with V2 image format
+ in the registry.
+
+- **`name`** *string*
+
+	name is the name of the image's repository
```

Comment:
@stevvooe is it worth mentioning here that this name will never include domain/server name of the registry or do you see that as a registry implementation detail.  This will get confusing later on when we use the fully qualified names in tags and attempt to support this schema version for backwards compatibility.